### PR TITLE
Fix broken link to kubernetes-csi/external-snapshotter/config/crd

### LINKED ISSUE #25399: Broken link in article https://kubernetes.io/blog/2019/12/09/kubernetes-1-17-feature-cis-volume-snapshot-beta/
Links are broken on this page:
https://kubernetes.io/blog/2019/12/09/kubernetes-1-17-feature-cis-volume-snapshot-beta/

### PR DESCRIPTION
**Pull request by bl-ue**
Fixes #25399

`/config/crd` was moved into `/client` in commit `kubernetes-csi/external-snapshotter@ae215218d1c47118bdc1a5abaf95dd8cfaf735d9`

### CODE REVIEW

File: content/en/blog/_posts/2019-12-09-volume-snapshot-beta.md
Comment (sftim) [inline content/en/blog/_posts/2019-12-09-volume-snapshot-beta.md:47]: Another option: link to the tree (by commit ID) at the time of the article. It's OK for blog articles to be a snapshot in time.

**Review by sftim — COMMENTED** [review]
/hold
(there's a Kubernetes release imminent - feel free to /unhold once that's gone out)
/lgtm

File: content/en/blog/_posts/2019-12-09-volume-snapshot-beta.md
Comment (bl-ue) [reply content/en/blog/_posts/2019-12-09-volume-snapshot-beta.md:47]: Great idea @sftim.

**Comment by bl-ue** [comment]
@sftim done.

**Comment by sftim** [comment]
/hold
(there's a Kubernetes release in progress - feel free to /unhold once that's _actually_ gone out)

**Comment by bl-ue** [comment]
Okay @sftim sorry about that!  How can I tell when it's "actually gone out"?

**Comment by bl-ue** [comment]
I thought that since [the v1.20.0 release](https://github.com/kubernetes/kubernetes/releases/tag/v1.20.0) has been created I can un-hold it.

**Comment by bl-ue** [comment]
Also @sftim what about [my other PRs](https://github.com/kubernetes/website/pulls/bl-ue)? Should they be on hold too? #25419 even has the `lgtm` and `approved` labels.

**Comment by sftim** [comment]
release_blocker_issue
Once that :x: from tide is gone, the deploy pipeline is open up again.

It's best to limit how many PRs are pushing against the floodgates for when we do open it up.

**Comment by bl-ue** [comment]
@sftim Okay, great.  Thank you.  I understand.

**Comment by bl-ue** [comment]
@sftim will you please add the lgtm and approved labels?

**Review by onlydole — APPROVED** [review]
Thank you, @bl-ue! 

/lgtm

**Pull request merged**
